### DISCUSSION TITLE
Dynamic Prompts - Wildcards often not working

### DISCUSSION THREAD

**Discussion by dhosein** [lllyasviel/stable-diffusion-webui-forge#1942] [Q&A]
I'm trying to use a set of wildcards to generate images and I'm finding a large proportion (over 50%) of the time, the wildcards are not being resolved. Sometimes it works.. most times it tries to generate an image based on the name of the wildcard. 
I'm using the ds-dynamic-prompts plugin. 
Any ideas what I'm doing wrong that makes the wildcards so unreliable?

**Comment by zeittresor** [comment]
hi there, i had a similar problem.. but find a little workaround by creating a script for this.. -> https://github.com/zeittresor/py_wildcard2prompt

**Comment by ricochoi** [comment]
I faced "sometimes work" problem,too...So why sometimes?
we type __clothes__ in promt and deploy some lora. For example, Morrimee style lora would regconize the "__" type string is a normal promt. So __chothes__ would not be affected on wildcards.

So you:
Go to Settings - Dynamic Promts,to find below option;
String to use as wrap for parser wildcard
And change __ to another string.I use $.
Save and reload the UI.
Change the __clothes__ to $clothes&.
And run!

**Reply by zeittresor** [reply]
hi there, interesting idea.. 🤗 i have not tested this yet but i will include a option for this into my style_going_wild project (https://github.com/zeittresor/style_going_wild) if that is working good.. 👍

**Comment by SillySilk** [comment]
I created a script that replaced._ with @@ Which, of course, none of my
files have. There is a place to actually change what the symbols are, but
they didn't work out of the box, and I needed to use Claude code to patch
it.
